src/NuGet.Clients/NuGet.CommandLine/NuGetCommand.resx

```diff
@@ -318,7 +318,7 @@ nuget pack foo.nuspec -Version 2.1.0</value>
     <value>Pushes a package to the server and publishes it.</value>
   </data>
   <data name="PushCommandSourceDescription" xml:space="preserve">
-    <value>Specifies the server URL. If not specified, nuget.org is used unless DefaultPushSource config value is set in the NuGet config file.</value>
+    <value>Specifies the server URL. This is a mandatory parameter unless the NuGet.Config file specifies a DefaultPushSource value. If NuGet identifies a UNC or local folder source, it simply copies the file there instead of pushing it using HTTP.</value>
```

Comment:
Just looking through old PRs and I noticed this one.

> If NuGet identifies a UNC or local folder source, it simply copies the file there instead of pushing it using HTTP.

If this means that the `nupkg` is copied to another folder and nothing else happens, I'm not sure that's correct. Doesn't NuGet detect the local folder source type (folder full of `nupkg`s, or `packageid.version\` folder with files extracted, or `packageid\version\` folder with files extracted) and do the right thing for each?

Personally, I think it would be better for this message to be short and generic, then on docs.microsoft/nuget have docs that explain more about HTTP vs local sources.

Reply:
```suggestion
    <value>Specifies the package source the package will be pushed to. This is a mandatory parameter unless the NuGet.Config file specifies a DefaultPushSource value.</value>
```

I'm afraid that I still don't like the message about a local source being a copy. It's an implementation detail that the user shouldn't need to worry about, and is overly simplistic to the point of being misleading, particularly in the case of V3 file feeds. After all, if it "simply copies the file there", then the push command isn't needed, the user could just copy the file themselves. The push command for local feeds is still valuable because it's more complicated than just copying a single file to another directory.

If it was added because the first sentence of the message said "URL", I propose we change it to be applicable to both HTTP and local feeds, so it's no longer necessary to call out local feeds.